### PR TITLE
IDE-679 ESDL Editor window should not offer Target choice

### PR DESCRIPTION
**Pull request by dehilsterlexis**
Fixed IDE-679

Signed-off-by: dehilsterlexis <david.dehilster@lexisnexis.com>

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

File: eclide/EclDlgBuilder.cpp [hunk 1]
```diff
@@ -265,6 +265,11 @@ LRESULT CBuilderDlg::OnInitDialog(HWND /*hWnd*/, LPARAM /*lParam*/)
     m_archiveButton = GetDlgItem(IDC_BUTTON_ARCHIVE);
 
     m_view.SetSourceType(m_comboQueueClusterCtrl->m_value);
+    if (m_attribute->GetType() == CreateIAttributeESDLType() || m_attribute->GetType() == CreateIAttributeECMType())
```
Comment (GordonSmith) [inline eclide/EclDlgBuilder.cpp]: @dehilsterlexis can you use the "new" way to test the type.

**Comment by dehilsterlexis** [comment]
@GordonSmith changed and added a null check. Back to you.

**Comment by GordonSmith** [comment]
Looks good.

**Pull request merged**